openslides_backend/action/actions/poll/create.py

```diff
@@ -0,0 +1,150 @@
+    "properties": {
+        "text": {"type": "string", "description": "the text of an option"},
+        "content_object_id": optional_fqid_schema,
+        "Y": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+        "N": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+        "A": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+    },
+    "additionalProperties": False,
+}
+
+
+@register_action("poll.create")
+class PollCreateAction(CreateAction):
+    """
+    Action to create a poll.
+    """
+
+    model = Poll()
+    schema = DefaultSchema(Poll()).get_create_schema(
+        required_properties=["title", "type", "pollmethod", "meeting_id"],
+        additional_required_fields={
+            "options": {
+                "type": "array",
+                "items": options_schema,
+                "minItems": 1,
+                "uniqueItems": True,
+            }
+        },
+        optional_properties=[
+            "content_object_id",
+            "description",
+            "min_votes_amount",
+            "max_votes_amount",
+            "global_yes",
+            "global_no",
+            "global_abstain",
+            "onehundred_percent_base",
+            "majority_method",
+            "votesvalid",
+            "votesinvalid",
+            "votescast",
+        ],
+        additional_optional_fields={
+            "amount_global_yes": {
+                "type": "string",
+                "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$",
+            },
+            "amount_global_no": {
+                "type": "string",
+                "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$",
+            },
+            "amount_global_abstain": {
+                "type": "string",
+                "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$",
+            },
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        payload = []
+
+        # handle non-global options
+        weight = 1
+        for option in instance.get("options", []):
+            data: Dict[str, Any] = {
+                "poll_id": instance["id"],
+                "meeting_id": instance["meeting_id"],
+                "weight": weight,
+            }
+            weight += 1
+            for key in ("text", "content_object_id"):
+                if key in option:
+                    data[key] = option[key]
+                if instance["type"] == "analog":
+                    if instance["pollmethod"] == "N":
+                        data["no"] = self.parse_vote_value(option, "N")
+                    else:
+                        data["yes"] = self.parse_vote_value(option, "Y")
+                        if instance["pollmethod"] in ("YN", "YNA"):
+                            data["no"] = self.parse_vote_value(option, "N")
+                        if instance["pollmethod"] == "YNA":
+                            data["abstain"] = self.parse_vote_value(option, "A")
+
+            payload.append(data)
```

Comment:
I think a check is missing here: If I create a poll with `pollmethod == "Y"` and give an option with `N == 1.000000`, the value will not be set on the option, but as far as I see it, it will just be passed through to `option.create`, so there will be a vote created with value `N` for a yes-poll.

Reply:
If pollmethod is "Y", data["no"]  is not set. and it is not transferred to option.create.

Reply:
Ah yes, my bad.